src/librustc_data_structures/graph/vec_graph/test.rs

```diff
@@ -0,0 +1,53 @@
+#![cfg(test)]
```

Comment:
Does this work the same as `#[cfg(test)] mod test;` in `mod.rs` for the purpose of dependency tracking? (cf #61097)

Reply:
No, the compiler needs to load and parse the file to discover the `#![cfg(test)]`, which puts it into the dependency list.